spec/rails_helper.rb

```diff
@@ -22,7 +22,11 @@
 
   config.include Rails.application.routes.url_helpers
 
+  config.include OAuthHelpers
+
   config.include Devise::Test::IntegrationHelpers, type: :system
 
   config.include DeviseHelpers::SystemHelpers
+
+  config.include OAuthHelpers
```

Comment:
Duplicated, we already added it before

Reply:
How couldn't I see it!! (0_o)' 
Thanks!